app/backend/requirements.txt

```diff
@@ -20,4 +20,4 @@ pyyaml
 requests
 sentry-sdk
 Shapely
-sqlalchemy<1.4
+sqlalchemy>=1.4
```

Comment:
I feel like this should be `<1.5`

Reply:
There are changes which will break on 1.3

Reply:
So maybe `>=1.4,<1.5` or something?

Reply:
So... =1.4? 😂

Reply:
This ought to also match `1.4.2`, etc

Reply:
It's the "half open interval [1.4,1.5)" as us mathematicians would call it :P